### PR TITLE
[blog] Flink Community Update - June'20

### PR DESCRIPTION
**Pull request by morsapaes**
Adding a Community Update blogpost for June 2020.

### CODE REVIEW

**Comment by rmetzger** [comment]
Thanks for opening this PR! It seems that the included image is quite large (1.5MB), given that it will stay forever in the git history. Can you compress it a bit?

**Comment by morsapaes** [comment]
Sorry, I didn't notice the file size, @rmetzger ! It should be better now.

Thanks for the quick reaction!

File: _posts/2020-06-10-community-update.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+As the community around StateFun grows, the release cycle will follow this pattern of smaller and more frequent releases to incorporate user feedback and allow for faster iteration. If you’d like to get involved, we’re always [looking for new contributors](https://github.com/apache/flink-statefun#contributing) — especially around SDKs for other languages (e.g. Go, Rust, Javascript).
+
+<hr>
+
+## Testing is ON for Flink 1.11
+
+Things have been pretty quiet in the Flink community, as all efforts shifted to testing the newest features shipping with Flink 1.11. While we wait for the first Release Candidate (RC) to be out, you can check the progress of testing in [this JIRA burndown board](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=364&projectKey=FLINK) and learn more about some of the [upcoming features](https://flink.apache.org/news/2020/05/07/community-update.html#warming-up-for-flink-111) in these Flink Forward videos:
```
Comment (rmetzger) [inline _posts/2020-06-10-community-update.md]: The first RC is out for quite a while: https://lists.apache.org/thread.html/r4952ccfd5f5562839f14b6e724f722b1302389c4b32c61db59ce9b31%40%3Cdev.flink.apache.org%3E

File: _posts/2020-06-10-community-update.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+### Flink 1.10.1
+
+The community released Flink 1.10.1, covering some outstanding bugs in Flink 1.10. You can find more in the [announcement blogpost](https://flink.apache.org/news/2020/05/12/release-1.10.1.html)!
+
+<hr>
+
+## New Committers and PMC Members
+
+The Apache Flink community has welcomed **2 new Committers** since the last update. Congratulations!
```
Comment (rmetzger) [inline _posts/2020-06-10-community-update.md:56]: Maybe mention their names?

File: _posts/2020-06-10-community-update.md (hunk 1 above)
Comment (morsapaes) [reply _posts/2020-06-10-community-update.md]: It's not a "true" RC in the sense of a precursor of the release, at least if I understand correctly. But I can amend it to "first voting RC" or "first RC to be out for voting", maybe?

**Review by rmetzger — APPROVED** [review]
The update looks fine.
I'm missing some "major technical updates" in here, but I guess that hasn't really happened because of the feature freeze-related testing / debugging.

**Comment by morsapaes** [comment]
I refrained from repeating what had already been written about the release in the previous update, but plan to start including feature discussions moving forward.

**Comment by rmetzger** [comment]
Thanks for creating the blog post!

**Pull request closed without merging**